optimism/src/mips/constraints.rs

```diff
@@ -48,6 +48,13 @@ impl<Fp: Field> InterpreterEnv for Env<Fp> {
 
     type Variable = Expr<ConstantExpr<Fp>, Column>;
 
+    fn variable(&self, column: Self::Position) -> Self::Variable {
```

Comment:
Technically could be a static function, used as `Env::variable(column)`?

Reply:
Yeah, it's just used to improve readability of the content of the functions inside the Env of constraints.